fiat/fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatAuthenticationFilter.java

```diff
@@ -43,13 +43,11 @@ public FiatAuthenticationFilter(
   @Override
   protected void doFilter(HttpServletRequest req, HttpServletResponse res, FilterChain chain)
       throws IOException, ServletException {
-    if (fiatStatus.isEnabled()) {
```

Comment:
If this is the right fix (and it seems reasonable that it is), I'd like to see a more specific test in addition to OperationsControllerIT.  Can you add tests with both fiatStatus enabled and disabled that use FiatAuthenticationFilter (and as little else as possible) and show that the user is present in the SecurityContext?

Reply:
Thanks David, that makes sense.
I will add focused tests around `FiatAuthenticationFilter` with `fiatStatus` both `enabled` and `disabled`, asserting that the authenticated user is correctly populated in the `SecurityContext`. These will be minimal, filter-level tests in addition to `OperationsControllerIT`.